data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/S3SelectCSVOption.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class S3SelectCSVOption {
+    static final String DEFAULT_CSV_HEADER = "USE";
+    @JsonProperty("file_header_info")
+    private String fileHeaderInfo = DEFAULT_CSV_HEADER;
+    @JsonProperty("quote_escape")
+    private String quiteEscape;
+    @JsonProperty("comments")
+    private String comments;
```

Comment:
Nit: Spacing between variables would make this easier to read. Applies to other pieces of this commit as well.

Reply:
Thanks Taylor for your review, Sure we will follow.